client/my-sites/people/controller.js

```diff
@@ -126,7 +126,7 @@ function renderSingleTeamMember( context ) {
 		user = UsersStore.getUserByLogin( siteId, userLogin );
 
 		if ( ! user ) {
-			UsersActions.fetchUser( { siteId: siteId }, userLogin );
+			UsersActions.fetchUserByLogin( { siteId: siteId }, userLogin );
```

Comment:
You could remove the object assignment here since the `key` has the same name that the var which has the value of the key (`siteId`).

```js
UsersActions.fetchUserByLogin( { siteId }, userLogin );
```

Again, it isn't a blocker.